Blish HUD/GameServices/Modules/ModuleManager.cs

```diff
@@ -38,6 +38,8 @@ public class ModuleManager : IDisposable {
         /// </summary>
         public bool Enabled { get; private set; }
 
+        public bool IsDisabling { get; private set; }
```

Comment:
These module changes probably shouldn't be in the PR and should probably have an explanation in the PR body.  Not sure the purpose here.

Reply:
Thanks, mate!